dependencies.yaml

```
@@ -563,6 +579,14 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - scikit-learn
+  java:
+    common:
+      - output_types: conda
+        packages:
+          - *cmake_ver
+          - maven
+          - openjdk=22.*
+          - libboost-devel
```

```suggestion
          - *cmake_ver
          - libboost-devel
          - maven
          - openjdk=22.*
```

Please keep these lists alphabetized.

---

fixed